backend/sirius-components-diagrams-tests/src/main/java/org/eclipse/sirius/components/diagrams/tests/builder/TargetObjectIdToNodeId.java

```diff
@@ -0,0 +1,37 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.components.diagrams.tests.builder;
+
+import java.util.HashMap;
+import java.util.UUID;
+
+/**
+ * The map used to store mapping between a target object id and its corresponding node id.
```

Comment:
Honestly I'm not convinced that you are using the good approach to solve this issue.

Reply:
Class removed